src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -79,3 +79,39 @@
     - name: Show the details for a connected kubernetes cluster
       text: az connectedk8s show -g resourceGroupName -n connectedClusterName
 """
+
+helps['connectedk8s proxy'] = """
+  type: command
+  short-summary: Get access to a connected kubernetes cluster.
+  examples:
+  - name: Get access to a connected kubernetes cluster.
+    text: az connectedk8s proxy -n clusterName -g resourceGroupName
+  - name: Get access to a connected kubernetes cluster with custom 'client proxy port' and 'api server port'
+    text: az connectedk8s proxy -n clusterName -g resourceGroupName --api-server-port portValue --client-proxy-port portValue
```

Comment:
>--api-server-port portValue [](start = 68, length = 27)

Why are we taking api server port?

Reply:
CSP uses two ports. "client proxy port" is an http port where we make register calls. "api server port" is an https port where it serves kubectl calls. We need to provide an override for "api server port" also in case the default one used by CSP is in use. Let me know if we should change the name of "api server port" to something else. For now, I am renaming "client proxy port" to just "port" as per your other comment.

Reply:
Renamed 'api server port' to 'port' after our discussion.